src/parser/expression.js

```diff
@@ -301,6 +301,24 @@ export default class ExpressionParser extends LValParser {
         node.object = base;
         node.callee = this.parseNoCallExpr();
         return this.parseSubscripts(this.finishNode(node, "BindExpression"), startPos, startLoc, noCalls);
+
+      } else if (this.eat(tt.questionDot)) {
```

Comment:
Similar to the other PR, we need to add a flag. I added the steps in https://github.com/babel/babylon/blob/master/CONTRIBUTING.md#creating-a-new-plugin-spec-new.

`hasPlugin('nullPropagation')` or `hasPlugin('optionalChaining')`, etc. Probably `optionalChaining` since the repo was changed to that.